spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 require 'axe/rspec'
 
-feature 'Accessibility on IDV pages', :js do
+feature 'Accessibility on IDV pages', :js, :accessibility do
```

Comment:
@monfresh it worked for me when I did a `--dry-run --format documentation` to check?